sdk-utils/src/test/kotlin/com/rakuten/tech/mobile/sdkutils/eventlogger/EventLoggerTestUtil.kt

```diff
@@ -0,0 +1,22 @@
+package com.rakuten.tech.mobile.sdkutils.eventlogger
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import kotlin.random.Random
+
+internal object EventLoggerTestUtil {
+
+    private val context: Context = ApplicationProvider.getApplicationContext()
+
+    fun generateRandomEvent(): Event {
+        val randomText = System.currentTimeMillis().toString()
+        return EventBuilder(context)
+            .buildEvent(
+                EventType.values()[Random.nextInt(EventType.values().size)],
+                randomText,
```

Comment:
Will we always use this **randomText** for sourceName, sourceVersion, code, message?

Reply:
@mdusmangani-jp, yes, just for test purposes, to not think about some dummy values. It's always possible to create with dummy values through `EventBuilder`.